domains/search.l6174.json

```diff
@@ -0,0 +1,11 @@
+{
+    "description": "My Self-Hosted Search Engine",
+    "repo": "https://github.com/l6174/search",
+    "owner": {
+        "username": "l6174",
+        "email": "l.lawliet6174@gmail.com"
+    },
+    "record": {
+        "CNAME": "8b8a7371-0cc1-4815-affb-f53446c7b444.id.repl.co"
```

Comment:
Denied as it has a similar name to Google.

Reply:
Bro, It isn't on my control. The developers choosed to keep that name. The repository which I cloned is named "search". You can checkout this repository https://github.com/benbusby/whoogle-search It has MIT License.

Reply:
@l6174 It's the name of the project, not who made named it.

Reply:
@WilliamDavidHarrison I don't get it? What I am where I am supposed to change the name? My cloned repository? Or replit repository?

Reply:
> @WilliamDavidHarrison I don't get it? Where I am supposed to change the name? My cloned repository? Or replit repository?

Where ever the website will show.

Reply:
> > @WilliamDavidHarrison I don't get it? Where I am supposed to change the name? My cloned repository? Or replit repository?
> 
> Where ever the website will show.

Nevermind, I am deleting search file. I will use https://search.l6174.repl.co .

Reply:
See this:
https://search.thomasc.is-a.dev/
It is using same "Whoogle" hosted on repl.
IDK Why I can't?

Reply:
I didn't merge that pull request. I'm not sure why it is there.

Reply:
Deleted. So, Basically we can't use whoogle with .is-a.dev. Anyways, I have deleted the file you can check further and merge. Thanks!

Reply:
If you want a domain for it, Free Domains will let you. https://github.com/free-domains/register